src/layouts/components/NavBar/GroupMenuBody.tsx

```diff
@@ -0,0 +1,164 @@
+                    <VStack contentEditable w="100%">
+                      <Editable.DraggableAccordionItem
+                        draggableId={`sublink-${index}-${sublinkIndex}-draggable`}
+                        index={sublinkIndex}
+                        title={sublink.title}
+                        isInvalid={_.some(errors.sublinks[sublinkIndex])}
+                      >
+                        <Editable.Section mt="-0.5rem">
+                          <FormControl
+                            isRequired
+                            isInvalid={!!errors.sublinks[sublinkIndex].title}
+                          >
+                            <FormLabel>Submenu Name</FormLabel>
+                            <Input
+                              placeholder="New menu group"
+                              id={`sublink-${index}-${sublinkIndex}-title`}
+                              value={sublink.title}
+                              onChange={onChange}
+                            />
+                            <FormErrorMessage>
+                              {errors.sublinks[sublinkIndex].title}
+                            </FormErrorMessage>
+                          </FormControl>
+                          <FormControl
+                            isRequired
+                            isInvalid={!!errors.sublinks[sublinkIndex].url}
+                          >
+                            <FormLabel>Destination</FormLabel>
+                            <Input
+                              placeholder="/permalink"
+                              id={`sublink-${index}-${sublinkIndex}-url`}
+                              value={sublink.url}
+                              onChange={onChange}
+                            />
+                            <FormErrorMessage>
+                              {errors.sublinks[sublinkIndex].url}
+                            </FormErrorMessage>
+                          </FormControl>
+                          <Button
+                            id={`section-${index}-${sublinkIndex}`}
+                            onClick={() =>
+                              onDelete(
+                                `sublink-${index}-${sublinkIndex}`,
+                                "Submenu"
+                              )
+                            }
+                            alignSelf="center"
+                            variant="clear"
+                            colorScheme="critical"
+                          >
+                            Delete submenu
+                          </Button>
+                        </Editable.Section>
+                      </Editable.DraggableAccordionItem>
+                    </VStack>
+                  ))}
+                </VStack>
+              </Editable.Accordion>
+            </Editable.EmptySection>
+          </Editable.Droppable>
+        </DragDropContext>
+      </FormControl>
+      <Box w="100%" pt="0.5rem">
```

Comment:
nit: non-functionality related, not important to fix but was wondering why not just pass the props on `Box` to the button directly

Reply:
Good point, fixed in https://github.com/isomerpages/isomercms-frontend/pull/1457/commits/40369daaa9268ba3779bf4604619b21a25a55100